projects/systelab-translate/src/lib/decimal-format/decimal-format.ts

```diff
@@ -111,7 +111,7 @@ export class DecimalFormat {
 			// toFixed method has bugs on IE (0.7 --> 0)
 			fracStr = num.toString(10)
 				.substr(2);
-			var c = (num >= 1) ? 1 : 0; //carry
+			var c = (Number(num) >= 1) ? 1 : 0; //carry
```

Comment:
Why is it needed this conversion? num is not implicy number? something new in new typescript version?

Reply:
It's a lint new error. num is not implicy number.